spec/invoker.e2e.spec.js

```diff
@@ -0,0 +1,131 @@
+const {TextEncoder} = require('util');
+const startInvoker = require('../lib/invoker');
+const outputSignalCustomEqual = require('./helpers/output-signal-custom-equality');
+const newClient = require('./helpers/grpc-client');
+const {
+    newInputFrame,
+    newInputSignal,
+    newOutputFrame,
+    newOutputSignal,
+    newStartFrame,
+    newStartSignal
+} = require('./helpers/factories');
+
+describe('invoker =>', () => {
+    const textEncoder = new TextEncoder();
+    let server;
+    let address;
+    let client;
+
+    afterEach((done) => {
+        server.tryShutdown(done);
+        client.close();
+    });
+
+    [
+        {interactionType: 'streaming', functionUri: '../samples/streaming-square/streaming-square.js'},
```

Comment:
Avoid using samples in tests for the invoker. People tend to modify samples without expecting that something important is depending on them. Testing the behavior of samples is good, but then I'd expect the test to cohabitate with the function in the sample dir.

Reply:
Copied the functions I needed to `spec/helpers/functions`